<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Model Fitter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QGridLayout" >
       <property name="spacing" >
        <number>1</number>
       </property>
       <item row="0" column="0" >
        <widget class="GLView" native="1" name="front" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="GLView" native="1" name="side" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="GLView" native="1" name="overhead" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="GLView" native="1" name="angle" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QGroupBox" name="meshGroup" >
         <property name="title" >
          <string>Mesh</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QListWidget" name="fileList" />
          </item>
          <item>
           <widget class="QPushButton" name="openButton" >
            <property name="text" >
             <string>Open directory...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="recalculateButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Recalculate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="recalculateAllButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Recalculate all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Cross-sections</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="crossFrontLabel" >
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Off</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSlider" name="crossFrontSlider" >
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="value" >
             <number>-1</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="crossSideLabel" >
            <property name="text" >
             <string>Off</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSlider" name="crossSideSlider" >
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="value" >
             <number>-1</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="crossTopLabel" >
            <property name="text" >
             <string>Off</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QSlider" name="crossTopSlider" >
            <property name="minimum" >
             <number>-1</number>
            </property>
            <property name="value" >
             <number>-1</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>View options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QCheckBox" name="showVoxelData" >
            <property name="text" >
             <string>Show voxel data</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showModel" >
            <property name="text" >
             <string>Show model</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Graph plotting</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" colspan="2" >
           <widget class="QComboBox" name="graphLimb" >
            <item>
             <property name="text" >
              <string>Left thigh</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Right thigh</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Left lower leg</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Right lower leg</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="graphDisplay" >
            <property name="text" >
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="graphSave" >
            <property name="text" >
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>238</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLView</class>
   <extends>QWidget</extends>
   <header>glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileList</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>recalculateButton</tabstop>
  <tabstop>recalculateAllButton</tabstop>
  <tabstop>crossFrontSlider</tabstop>
  <tabstop>crossSideSlider</tabstop>
  <tabstop>crossTopSlider</tabstop>
  <tabstop>showVoxelData</tabstop>
  <tabstop>showModel</tabstop>
  <tabstop>graphSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
